### DISCUSSION TITLE
Is there no click event for SliderMark?

### DISCUSSION THREAD

**Discussion by Abcold** [chakra-ui/chakra-ui#6904] [Q&A]
I tried to add click events in SliderMark, but it didn't work. Can you provide an example？
`  <Slider aria-label="slider-ex-6" onChange={(val) => setSliderValue(val)}>

                 <SliderMark
                    className={noto_t3}
                    value={75}
                    zIndex="1"
                    onClick={() => {
                        console.log('75');
                    }}
                    {...labelStyles}
                >
                    <Circle
                        w="10px"
                        h="10px"
                        bg="#FFFFFF"
                        border={sliderValue >= 75 ? '2px solid #54AB6B' : '2px solid #B3B3B3'}
                        borderRadius="50%"
                        mt="-13px"
                        onClick={() => {
                            console.log('75');
                        }}
                    ></Circle>
                </SliderMark>
 <SliderTrack bg="linear-gradient(90deg, #F1F1F1 -6.17%, #F2F2F2 102.06%)">
                    <SliderFilledTrack bg="#12931F" />
                </SliderTrack>
                <SliderThumb w="12px" h="12px" _focus={{}}>
                    <Circle w="12px" h="12px" bg="#FFFFFF" border="2px solid #12931F" borderRadius="50%"></Circle>
                </SliderThumb>
            </Slider> `

**Comment by joekrill** [comment]
I, too, was trying to make them clickable. The [SliderMarker props have `pointerEvents: "none"` hard-coded](https://github.com/chakra-ui/chakra-ui/blob/6d12ea81f4791f4df2953533b570178f59551b74/packages/components/slider/src/use-slider.ts#L497). You could can set `pointerEvents="auto"` on your `Circle` component, but that causes some additional UI issues because the Slider starts picking up your click events. I'm guessing this is why it was simply disabled for now by settings `pointerEvents` to "none"?
